lib/crypto/crypto_keys_subr.c

```diff
@@ -32,7 +32,7 @@
  */
 
 static int import_BN(BIGNUM **, const uint8_t **, size_t *);
-static int export_BN(BIGNUM *, uint8_t **, size_t *, uint32_t *);
+static int export_BN(const BIGNUM *, uint8_t **, size_t *, uint32_t *);
```

Comment:
I'm confused.  OpenSSL 0.9.6 certainly uses `const BIGNUM *` for `BN_bn2bin()` [1], so what's the fuss?  I mean, if we ever switch to a different library that has a `BN_bn2bin` which takes a non-const value, couldn't we change at that time?

[1] https://github.com/openssl/openssl/blob/OpenSSL_0_9_6-stable/crypto/bn/bn.h#L344